src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Scripts/Sequences.sql

```diff
@@ -59,3 +59,11 @@ CREATE SEQUENCE dbo.WorkitemKeySequence
     MINVALUE 1
     NO CYCLE
     CACHE 10000
+
+CREATE SEQUENCE dbo.WorkitemWatermarkSequence
+    AS BIGINT
+    START WITH 1
+    INCREMENT BY 1
+    MINVALUE 1
+    NO CYCLE
+    CACHE 1000000
```

Comment:
caching 10000 is fine since we are caching only 10000 for workitem key

Reply:
Done